ttnn/cpp/ttnn/operations/sliding_window/sliding_window.cpp

```diff
@@ -24,6 +25,14 @@ bool SlidingWindowConfig::has_parallel_config() const {
     * Calculate the window op output shape, excludes the channel dimension since this config is independent of the depth.
     */
 Shape SlidingWindowConfig::get_output_shape() const {
+    if(is_transpose) {
+        //Inverse of is_transpose == False
```

Comment:
this comment needed?

Reply:
I meant that it does the inverse calculation of the forward pass.

Reply:
Wrote a better comment.